sos/plugins/kernel.py

```diff
@@ -65,6 +77,8 @@ def setup(self):
             "/sys/module/*/refcnt",
             "/sys/module/*/taint",
             "/sys/firmware/acpi/*",
+            "/sys/kernel/debug/tracing/*",
+            "/sys/kernel/debug/sunrpc/*",
```

Comment:
Wouldn't this be better in the `sunrpc` plugin?

Reply:
(which may need some work on its `check_enabled()`...)

Reply:
Yup, you're right. I totally forgot that was a thing. I'll remove this bit later today.

Reply:
Cool 👍 

I think we all did a bit.. It doesn't seem to have had much lovin' in a while...